setup.py

```diff
@@ -15,7 +15,11 @@
     author_email="dev-support@authy.com",
     url="http://github.com/authy/authy-python",
     keywords=["authy", "two factor", "authentication"],
-    install_requires=["requests>=2.2.1", "simplejson>=3.4.0", "six>=1.8.0"],
+    install_requires=[
+        "requests>=2.2.1",
+        "six>=1.8.0",
+        "simplejson>=3.4.0;python_version<'2.6'"
+    ]
```

Comment:
missing a comma here.

I'm going to verify all the places simplejson is used before I will merge this. we still have import statements and a requirements.txt file that reference it

Reply:
Hi @robinske 👋 

I have added the missing comma as requested. Doh! 🤦 
I did already check the usage of that lib and all seems well - the code seems to always try to import from the core python language first and then falls back to ``simplejson`` and failing that it tries to load ``django``'s json module. 👍